architecture/minimum_viable_product.md

```diff
@@ -0,0 +1,41 @@
+# Minimum Viable Product.
+The mvp should be so simple as it should be. There are so many alternatives.
+In my idea, The minimum viable product is as follows
+
+## Server
+Backend server should be capable of recieving a request with current location from end user. Response of the request is list of auto rickshaws around current location. 
+
+Another facility needed is  registering auto rickshaw cooperative unions and auto rickshaw drivers.
+Drivers will register under a cooperative union and cooperative union will verify the driver.
+Driver rating is another capability needed for MVP.
```

Comment:
I mentioned Cooperative union as a group, privileged for handling drivers and rides in the specified territory.
we haven’t implemented distributed computation. So there will be a centralized server. So the coop is not hosting the service. they receive service from LibreCoop India